File-Formats-toc.html

```diff
@@ -494,6 +494,7 @@
 					<li><a href="/file-formats/xlsio/faqs/how-to-open-an-excel-document-that-is-already-open-in-msexcel">How to open an Excel document using XlsIO that is already open in MS-Excel?</a></li>
 					<li><a href="/file-formats/xlsio/faqs/reason-for-parameter-not-valid-exception-when-converting-exceltopdf-with-custompapersize">Reason for Parameter Not Valid exception when converting Excel to PDF with Custom Paper Size</a></li>
 					<li><a href="/file-formats/xlsio/faqs/information-about-maximum-characters-count-in-an-excel-cell">Information about maximum characters count in an Excel cell</a></li>
+					<li><a href="/file-formats/xlsio/faqs/how-to-search-for-a-value-in-different-columns-in-an-Excel-worksheet">How to search for a value in different columns in an Excel worksheet?</a></li>
```

Comment:
@SivakumarRamya - Please change the title to "How to search a value in only specific columns of an Excel worksheet"